### ISSUE TITLE
Power Pole SMART placing

### ISSUE THREAD

**Issue by snowboundmage2** [SmartFoundations/SmartIssueTracker#70]
I understand power connection/lines is an issue, but what about just the poles themselves? I would love it if poles could be placed with smart, even if they have to be manually wired up.

Making a super long foundation is great, and placing stackable conveyors along it is also great, but what if I want to do some long distance power coverage without trains? Or just building neatly placed poles along a really busy bus? 

Spacing them out to match my conveyor bus stackable poles would be so much neater then manually placing poles anywhere and everywhere they're needed..

My workflow for neatly powering up busses is running along (can't even glide due to no power being placed yet), having to stop, precisely check the poles placement, then place it, then connect it to the previous pole, then repeat the whole thing again.
Could be simplified by preparing a pole placement, and spacing them exactly the meters I want.

My workflow for travelling long distances is connect to a previous pole, slide jump with my crosshair aimed at the path until the pole turns red, slowing down and placing the pole, connecting to the newly placed pole and then repeating this.
Could be simplified by preparing a pole placement, and spacing the poles onto forever with a max spacing of 100m(max connection length), or 96m(max connection length neatly divisible by foundation length/width).

**Comment by SmartFoundations** [comment]
@snowboundmage2  But you understand that when travelling(exploring) and placing them in one line they can be placed inside trees, ground or mountains? So that will only work when building on a foundation

Also - this was a bad idea before  (because how could you know when to stop placing poles if they are 100m apart and you build many off them) now with the camera function you can control that much better! So I will take that into consideration

**Comment by snowboundmage2** [comment]
Yeah, fair enough, I'm just thinking of all the sky busses I (and probably many other new players) built in the past. But it also was also recently, when trying to even out my base I just got bored real quick of placing poles every 48m long, and 3m to the left of center. 
And the camera is beautiful, I've always noted how long paths are and how much spacing is on a certain thing, now the camera really helps in that regard.

**Comment by majormer** [comment]
This issue has been migrated to the new repository.

**New location:** https://github.com/majormer/SmartFoundations/issues/69

Please follow up there.
